src/Symfony/Bundle/SecurityBundle/Routing/LogoutRouteLoader.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\Routing;
+
+use Symfony\Component\DependencyInjection\Config\ContainerParametersResource;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+class LogoutRouteLoader
+{
+    /**
+     * @param iterable<string, string> $logoutPaths   Logout paths indexed by the corresponding firewall name
```

Comment:
looking at the usage in ContainerParametersResource, does this really support any iterables or only arrays ? Container parameters don't allow objects.

Reply:
I forgot to change the type when trying the `ContainerParametersResourceChecker` :see_no_evil: there is no need to hint `$logoutPaths` as iterable